### DISCUSSION TITLE
Set depth for Exception analysis

### DISCUSSION THREAD

**Discussion by andrii-pez** [JetBrains/Qodana#111] [Q&A] [answered]
Cheers! 
Is it possible to set depth for exception analysis, just like it's done in PhpStorm?
![Screenshot 2022-07-29 at 14 18 20](https://user-images.githubusercontent.com/56593990/181748245-7458f4ad-ab34-4a42-b30c-c27aa1268439.png)

**Comment by brichbash** [comment, marked as answer]
Hi!
The information about the depth is stored in `<your-project>/.idea/php.xml`, looks like this:
```
<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="PhpAnalysisConfiguration" call_tree_analysis_depth="2" />
...
</project>
```
Qodana automatically uses configuration files from `.idea`, so if you have/create/move `.idea/php.xml` before analysis in your project root in any CI/CD, Qodana should consider the settings.
